exercises/leap/src/test/java/LeapTest.java

```diff
@@ -0,0 +1,61 @@
+import org.junit.Ignore;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class LeapTest {
+
```

Comment:
Please use object based approach as mentioned in the comment concerning the reference implementation.

Reply:
Done.